### PR TITLE
MGMT-11361 Storage integration tests

### PR DESCRIPTION
**Pull request by celdrake**
https://issues.redhat.com/browse/MGMT-11361

Integration tests for Storage step in Assisted Installer Wizard

### CODE REVIEW

**Comment by celdrake** [comment]
My fork had the incorrect base repo. I've redone it linked to the correct one (https://github.com/openshift-assisted/openshift-assisted-ui)

File: packages/integration-tests/src/fixtures/hosts/index.ts [hunk 1]
```diff
@@ -7,10 +7,13 @@ const hostIds = [
...
   '3e40aa5d-0b69-4122-a562-bff1e35e7073',
+  '4e40aa5d-0b69-4122-a562-bff1e35e7074',
+  '5e40aa5d-0b69-4122-a562-bff1e35e7075',
 ];
-const hostIPs = ['192.168.122.0/24', 'IP2', 'IP3'];
 
-const discoveredHosts = [hostDiscover(0), hostDiscover(1), hostDiscover(2)];
+const hostIPs = ['192.168.122.0/24', 'IP2', 'IP3', 'IP4', 'IP5'];
+
+const discoveredHosts = [hostDiscover(0), hostDiscover(1), hostDiscover(2), hostDiscover(3), hostDiscover(4)];
```
Comment (jkilzi) [inline packages/integration-tests/src/fixtures/hosts/index.ts:16]:
```suggestion
const discoveredHosts = Array(5).fill(hostDiscover).map((f, idx) => f(idx));
```
:)~

**Review by jkilzi — APPROVED** [review]

**Review by ammont82 — APPROVED** [review]
Maybe we need to add the new option of "Skip formatting" in bootable disks in the storage integration tests for version 2.11?
Otherwise LGTM

**Comment by celdrake** [comment]
> Maybe we need to add the new option of "Skip formatting" in bootable disks in the storage integration tests for version 2.11? Otherwise LGTM

Yes, it's always good to improve the coverage. We can keep this in mind in case there is extra capacity in the Sprint

File: packages/integration-tests/src/fixtures/hosts/index.ts (hunk 1 above)
Comment (celdrake) [reply packages/integration-tests/src/fixtures/hosts/index.ts:16]: Did a slight variation, but same idea to not DRY.

Comment (celdrake) [reply packages/integration-tests/src/fixtures/hosts/index.ts:16]: For some reason it wasn't updating this branch, I added it to Read Only cluster story
https://github.com/openshift-assisted/openshift-assisted-ui/pull/18/commits/336754bb39fa31b8d64daba7c48ae895d704b60a

**Pull request merged**